ariadne/cardano/src/Ariadne/Cardano/Knit.hs

```diff
@@ -16,6 +16,8 @@ module Ariadne.Cardano.Knit
        , ComponentCommandExec(..)
        , ComponentCommandProcs(..)
 
+       , adaToCoin
```

Comment:
Well, instead of exporting a helper function I would test `tyCoin` instead. `TyProjection` has `tpMatcher`, so it should be very similar to what you did. Such a test will also prevent silent changes of `Coin` representation.

Reply:
https://github.com/serokell/ariadne/pull/372/files#diff-a80eb03e3e64bd5e2e9ba75af1effb84R105

Reply:
Ok then :man_shrugging: